packages/big-design/src/components/List/Item/useContent.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { cloneElement, isValidElement, useCallback, useMemo } from 'react';
+
+import { DropdownItem, DropdownLinkItem } from '../../Dropdown';
+import { Flex, FlexItem } from '../../Flex';
+import { SelectAction, SelectOption } from '../../Select';
+import { Tooltip } from '../../Tooltip';
+import { Small } from '../../Typography';
+
+import { StyledLink } from './styled';
+
+interface useContentProps {
+  item: DropdownItem | DropdownLinkItem | SelectOption<any> | SelectAction;
+  isHighlighted: boolean;
+}
+
+export const useContent = ({ item, isHighlighted }: useContentProps) => {
+  const iconColor = useMemo(() => {
+    if (item.disabled) {
+      return 'secondary40';
+    }
+
+    if (!isHighlighted || !('onActionClick' in item)) {
+      return 'secondary60';
+    }
+
+    return 'actionType' in item ? (item.actionType === 'destructive' ? 'danger50' : 'primary') : 'primary';
```

Comment:
🍹 it could be simpler 👍 
```suggestion
    return 'actionType' in item && item.actionType === 'destructive' ? 'danger50' : 'primary';
```